engine/src/main/java/io/zeebe/engine/nwe/event/StartEventProcessor.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.0. You may not use this file
+ * except in compliance with the Zeebe Community License 1.0.
+ */
+package io.zeebe.engine.nwe.event;
+
+import io.zeebe.engine.nwe.BpmnElementContext;
+import io.zeebe.engine.nwe.BpmnElementProcessor;
+import io.zeebe.engine.nwe.behavior.BpmnBehaviors;
+import io.zeebe.engine.nwe.behavior.BpmnEventSubscriptionBehavior;
+import io.zeebe.engine.nwe.behavior.BpmnIncidentBehavior;
+import io.zeebe.engine.nwe.behavior.BpmnStateBehavior;
+import io.zeebe.engine.nwe.behavior.BpmnStateTransitionBehavior;
+import io.zeebe.engine.nwe.behavior.BpmnVariableMappingBehavior;
+import io.zeebe.engine.processor.workflow.deployment.model.element.ExecutableStartEvent;
+
+public class StartEventProcessor implements BpmnElementProcessor<ExecutableStartEvent> {
+
+  private final BpmnEventSubscriptionBehavior eventSubscriptionBehavior;
+  private final BpmnStateTransitionBehavior stateTransitionBehavior;
+  private final BpmnVariableMappingBehavior variableMappingBehavior;
+  private final BpmnIncidentBehavior incidentBehavior;
+  private final BpmnStateBehavior stateBehavior;
+
+  public StartEventProcessor(final BpmnBehaviors bpmnBehaviors) {
+    eventSubscriptionBehavior = bpmnBehaviors.eventSubscriptionBehavior();
+    incidentBehavior = bpmnBehaviors.incidentBehavior();
+    stateBehavior = bpmnBehaviors.stateBehavior();
+    stateTransitionBehavior = bpmnBehaviors.stateTransitionBehavior();
+    variableMappingBehavior = bpmnBehaviors.variableMappingBehavior();
+  }
+
+  @Override
+  public Class<ExecutableStartEvent> getType() {
+    return ExecutableStartEvent.class;
+  }
+
+  @Override
+  public void onActivating(final ExecutableStartEvent element, final BpmnElementContext context) {
+    // todo(@korthout): discuss the input mappings, there are no tests for this
```

Comment:
I think input mappings on a start event doesn't make sense. A start event is activated on workflow instance creation or when it is triggered by an event. So, I see no use case of an input mapping - only for output mappings. 
Let's remove it if you agree.